src/spinner.vue

```diff
@@ -1,23 +1,24 @@
 <template lang="jade">
-.mdl-spinner.mdl-js-spinner(v-bind:class='{ "mdl-spinner--single-color": singleColor, "is-active": active }')
+.mdl-spinner.mdl-js-spinner(v-bind:class="{ 'mdl-spinner--single-color': singleColor, 'is-active': active, 'is-upgraded': upgraded }")
```

Comment:
The initial call to upgrade in the mounted lifecycle callback works just as expected. However, the first time you change any property that causes the binding on `class` to be recalculated, the 'is-upgraded' the MDL itself added dynamically gets removed by vue. AFAICT, it's basically the same problem reported here: http://forum.vuejs.org/topic/3963/keep-dynamically-added-class-after-re-rendering/2

I'm happy to remove it, but many of the components fail to render properly after interacting with them because 'is-upgraded' gets dropped during the vue class binding update.